File: mobile/src/main/java/com/ataulm/whatsnext/api/ApiMemberAccountResponse.java
package com.ataulm.whatsnext.api;

import com.google.gson.annotations.SerializedName;

class ApiMemberAccountResponse {

    @SerializedName("member")
    Member member;

    static class Member {
        @SerializedName("id")
        String letterboxId;

        @SerializedName("displayName")
        String displayName;
    }
}

File: app/src/main/java/com/ataulm/whatsnext/api/ApiMemberAccountResponse.kt
package com.ataulm.whatsnext.api

import com.google.gson.annotations.SerializedName

class ApiMemberAccountResponse {

    @SerializedName("member")
    var member: Member? = null

    class Member {
        @SerializedName("id")
        var letterboxId: String? = null

        @SerializedName("displayName")
        var displayName: String? = null
    }
}
